### PR TITLE
Update release instructions

### LINKED ISSUE #1942: Documentation update: Updating the steps to cut a release of PHDI: https://github.com/CDCgov/phdi/blob/main/RELEASE.md

### PR DESCRIPTION
**Pull request by bamader**
# Update release instructions

## Summary
This PR updates our RELEASE.md file to show the actual steps involved in cutting a new release and deploying it, rather than the outdated and cumbersome instructions that were there.

## Related Issue
Fixes #1942

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -34,61 +34,32 @@ API documentation is published automatically with Sphinx and hosted via GitHub p
...
+### Create a Release Branch
+Once the new version number is defined and all desired functionality is merged into the `main` branch of the github repository, you can safely begin executing the release process. First, create a new branch off of `main` called `release-MAJOR.MINOR.PATCH` (replace each of the SemVer components with the appropriate number in the new release version).
 
-#### Create a GitHub Release
-A GitHub release ties together a few pieces of information in the release process:
-1) A tag for the commit that comprises the release
-2) A short description/name for the release
-3) A list of changes in the release
+### Update Versioning Files
+On the release branch, navigate to the `phdi/__init__.py` file. Replace the version numebr there with the new release's version number. Then, navigate to the `pyproject.toml` file and do the same thing with the `version` keyword on line 3.
```
Comment (robertandremitchell) [inline RELEASE.md:51]: number

**Review by robertandremitchell — APPROVED** [review]
looks good, one minor spelling thing.

File: RELEASE.md (hunk 1 above)
Comment (bamader) [reply RELEASE.md:51]: lol classic

**Review by m-goggins — APPROVED** [review]
Looks good! The only suggestion I would make is to update the release [tutorial ](https://github.com/CDCgov/phdi/blob/main/tutorials/release-tutorial.md)to have the same info or delete the tutorial.

**Comment by bamader** [comment]
Can do, I'll delete the tutorial since this one is more comprehensive now

**Pull request merged**